frontend/app/src/components/calendar/CalendarReminderEntry.vue

```diff
@@ -0,0 +1,214 @@
+<script setup lang="ts">
+import { helpers, maxValue, minValue, required } from '@vuelidate/validators';
+import useVuelidate from '@vuelidate/core';
+import { toMessages } from '@/utils/validation';
+import type { CalendarReminderTemporaryPayload } from '@/types/history/calendar/reminder';
+
+const props = defineProps<{
+  value: CalendarReminderTemporaryPayload;
+}>();
+
+const emit = defineEmits<{
+  (e: 'input', value: number): void;
+  (e: 'delete'): void;
+}>();
+
+const { value } = toRefs(props);
```

Comment:
Any change we can wrap the v-model with the utility we have here and us the v-model further down?
It will make stuff easier for vue 3 otherwise all the references of value have to be manually changed to modelValue